src/app/Board/index.ts

```diff
@@ -0,0 +1,53 @@
+import './board.css'
+import { appContainer } from '../constants';
+import { Renderable } from '../types/renderable.interface';
+import { Tile } from '../Tile';
+
+export class Board implements Renderable {
+  private readonly tiles: Array<Array<Tile>> = [];
+  readonly container = document.createElement('div');
+  readonly rows: Array<HTMLDivElement> = [];
+  constructor(
+    private readonly xSize: number,
+    private readonly ySize: number,
+  ) {
+    this.initialize()
+  }
+
+  private initializeRows() {
+    for(let x = 0; x < this.xSize; x++) {
+      const row = document.createElement('div');
+      row.classList.add('row')
+      this.rows[x] = row;
+      this.container.append(row);
+    }
+  }
+  private initialize() {
+    appContainer!.append(this.container);
```

Comment:
Using `!` is a bad practice, it's almost as bad as using `any` because you are overriding TypeScript types.